src/pages/release-announce/1.5.1.md

```diff
@@ -0,0 +1,49 @@
+---
+id: rn-1.5.1
+title: Release Announcement v1.5.1
+---
+
+<!--
+Licensed to the Apache Software Foundation (ASF) under one
+or more contributor license agreements.  See the NOTICE file
+distributed with this work for additional information
+regarding copyright ownership.  The ASF licenses this file
+to you under the Apache License, Version 2.0 (the
+"License"); you may not use this file except in compliance
+with the License.  You may obtain a copy of the License at
+
+  http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing,
+software distributed under the License is distributed on an
+"AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+KIND, either express or implied.  See the License for the
+specific language governing permissions and limitations
+under the License.
+-->
+
+# Release Announcement v1.5.1
+We are pleased to announce that the Apache YuniKorn community has voted to release 1.5.1. 
+Apache YuniKorn is a standalone resource scheduler, designed for managing and scheduling Batch and Data workloads on container
+orchestration frameworks like Kubernetes for on-prem and on-cloud use cases.
+
+## Overview
+The Apache YuniKorn community has resolved 18 [JIRAs](https://issues.apache.org/jira/issues/?filter=12353383) in this release.
+
+Release manager: Peter Bacsko
+
+Release date: 2024-05-15
+
+## Highlights
+
+### Locking fixes to avoid existing and potential deadlocks
+
+YuniKorn 1.5.0 contained a deadlock which is now fixed in 1.5.1 along with other potential
+deadlocks. See [YUNIKORN-2553](https://issues.apache.org/jira/browse/YUNIKORN-2553) for details.
```

Comment:
Change to:

YuniKorn 1.5.0 contained a deadlock as described in [YUNIKORN-2521](https://issues.apache.org/jira/browse/YUNIKORN-2521). This deadlock is now fixed along with other potential deadlocks. See [YUNIKORN-2544](https://issues.apache.org/jira/browse/YUNIKORN-2544) for a more detailed overview of the locking changes made.

Reply:
Done.